src/style.css

```diff
@@ -1,3 +1,51 @@
+html {
+  font-family:
+    system-ui,
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    Roboto,
+    Oxygen,
+    Ubuntu,
+    Cantarell,
+    'Open Sans',
+    'Helvetica Neue',
+    sans-serif;
+}
+
 body {
   margin: 0;
+  display: flex;
+  width: 40px;
+  height: 40px;
```

Comment:
Setting fixed width and height on the `body` element is not recommended as it can lead to layout issues, especially on different screen sizes. Consider using relative units or removing these properties for better responsiveness.